### PR TITLE
Dominian Loadout Additions

### PR DESCRIPTION
**Pull request by Catrly**
BWUH. I'm so sorry for the gitlog spam. I'm going to just not touch it once this is up. No idea what's going wrong.

~~Nonetheless, adds the NCO patch and two dominian cloaks into the loadout.~~ 

Adds a dominian cape to the loadout. Requested by Schwann.

### CODE REVIEW

**Comment by Catrly** [comment]
!review

File: code/modules/client/preference_setup/loadout/items/suit.dm [hunk 1]
```diff
@@ -291,6 +291,8 @@
 	dominiacape["white dominian cape, caladius"] = /obj/item/clothing/accessory/poncho/dominia_cape/caladius/white
 	dominiacape["dominian cape, zhao"] = /obj/item/clothing/accessory/poncho/dominia_cape/zhao
 	dominiacape["white dominian cape, zhao"] = /obj/item/clothing/accessory/poncho/dominia_cape/zhao/white
+	dominiacape["dominian cape, military"]	= /obj/item/clothing/accessory/poncho/dominia_cape/mantle
+	dominiacape["dominian cape, medical"] = /obj/item/clothing/accessory/poncho/dominia_cape/hospital
```
Comment (DreamySkrell) [inline code/modules/client/preference_setup/loadout/items/suit.dm:295]: image
this already exists in the loadout

File: code/modules/client/preference_setup/loadout/items/suit.dm [hunk 2]
```diff
@@ -291,6 +291,8 @@
 	dominiacape["white dominian cape, caladius"] = /obj/item/clothing/accessory/poncho/dominia_cape/caladius/white
 	dominiacape["dominian cape, zhao"] = /obj/item/clothing/accessory/poncho/dominia_cape/zhao
 	dominiacape["white dominian cape, zhao"] = /obj/item/clothing/accessory/poncho/dominia_cape/zhao/white
+	dominiacape["dominian cape, military"]	= /obj/item/clothing/accessory/poncho/dominia_cape/mantle
```
Comment (DreamySkrell) [inline code/modules/client/preference_setup/loadout/items/suit.dm:294]: from the item's desc

> Commonly worn by enlisted soldiers of the Imperial military
> Many armsmen carry on this trend in order to maintain their Imperial Army roots

File: code/modules/client/preference_setup/loadout/items/accessories.dm [hunk 3]
```diff
@@ -703,3 +703,9 @@
 	display_name = "visegradi patterned sweater"
 	path = /obj/item/clothing/accessory/sweater/visegradi
 	flags = GEAR_HAS_DESC_SELECTION | GEAR_HAS_COLOR_SELECTION
+
+/datum/gear/accessory/nco_badge
+	display_name = "imperial army badge"
+	path = /obj/item/clothing/accessory/dominia/nco_badge
+	flags = GEAR_HAS_DESC_SELECTION
+	culture_restriction = list(/singleton/origin_item/culture/dominia, /singleton/origin_item/culture/dominian_unathi)
```
Comment (DreamySkrell) [inline code/modules/client/preference_setup/loadout/items/accessories.dm:711]: item's desc
> A sleeve patch worn by non-commissioned officers of the Imperial Army.

I really do not like the idea of adding actual military items to the loadout.

General military items should be fine, stuff like the TCFL jackets, or gadpathur things. But not actual military badges, that show the character holding some rank or whatever.

Horizon crew is working for SCC or the other corps. They should not be holding ranks, as if they were still in active duty, and should not be parading around a corporate ship wearing badges or insignia with these ranks.

**Review by DreamySkrell — CHANGES_REQUESTED** [review]

File: code/modules/client/preference_setup/loadout/items/accessories.dm (hunk 3 above)
Comment (Catrly) [reply code/modules/client/preference_setup/loadout/items/accessories.dm]: What change is it you want here, sorry?

I don't disagree regarding the patch - was asked to put it in so here I am. Happy to remove it + the medical cape. Not following what you need for the other one.

File: code/modules/client/preference_setup/loadout/items/suit.dm (hunk 2 above)
Comment (Catrly) [reply code/modules/client/preference_setup/loadout/items/suit.dm:294]: What is it you're asking for here, sorry?

Comment (DreamySkrell) [reply code/modules/client/preference_setup/loadout/items/suit.dm:294]: this was just to highlight that the same point applies to it as to about the NCO patch

but I think this cape should be fine-ish, cause it's not explicitly like a ranked officer's cape or something, but just a general "this is what armsmen sometimes wear" cape (at least that's how I understand it)

File: code/modules/client/preference_setup/loadout/items/accessories.dm (hunk 3 above)
Comment (DreamySkrell) [reply code/modules/client/preference_setup/loadout/items/accessories.dm]: I just wanted the NCO patch removed from the PR, yeah

**Review by DreamySkrell — APPROVED** [review]
looks fine now, but do update changelog and PR desc though

**Review by FluffyGhoster — APPROVED** [review]

**Pull request merged**